app/_locales/en/messages.json

```diff
@@ -2161,6 +2173,9 @@
   "imToken": {
     "message": "imToken"
   },
+  "immediateAccessToYourTokens": {
+    "message": "Immediately access to your tokens"
+  },
```

Comment:
Should this say "Immediately access your tokens" Or "Immediate access to your tokens" instead of "Immediately access to your tokens" ?